Library/Formula/osquery.rb

```diff
@@ -43,12 +42,9 @@ def install
                                  "--record=installed.txt"}
     end
 
-    mkdir "generated"
     system "cmake", ".", *std_cmake_args
     system "make"
     system "make", "install"
-
-    prefix.install "tools/deployment/com.facebook.osqueryd.plist"
```

Comment:
`make install` installs it, yup: https://github.com/facebook/osquery/blob/master/osquery/CMakeLists.txt#L129-L130